### PR TITLE
[feature](mtmv)MTMV pause and resume

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

- PAUSE MATERIALIZED VIEW JOB ON mv1
- RESUME MATERIALIZED VIEW JOB ON mv1
- fix when drop db,not drop job
- add lock for one materialized view can only run one task at a time

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/PauseMTMVCommand.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    public PauseMTMVCommand(PauseMTMVInfo pauseMTMVInfo) {
+        super(PlanType.PAUSE_MTMV_COMMAND);
+        this.pauseMTMVInfo = Objects.requireNonNull(pauseMTMVInfo, "require pauseMTMVInfo object");
+    }
+
+    @Override
+    public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        pauseMTMVInfo.analyze(ctx);
+        Env.getCurrentEnv().getMtmvService().pauseMTMV(pauseMTMVInfo);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/PauseMTMVCommand.java:43]: Where to check the auth?

Comment (zddr) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/PauseMTMVCommand.java:43]: pauseMTMVInfo.analyze(ctx);

**Comment by zddr** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by seawinde — APPROVED** [review]

**Pull request merged**